TESTS/netsocket/udp/main.cpp

```diff
@@ -180,10 +191,14 @@ Case cases[] = {
     Case("UDPSOCKET_BIND_UNOPENED", UDPSOCKET_BIND_UNOPENED),
     Case("UDPSOCKET_SENDTO_INVALID", UDPSOCKET_SENDTO_INVALID),
     Case("UDPSOCKET_ECHOTEST_NONBLOCK", UDPSOCKET_ECHOTEST_NONBLOCK),
+#if MBED_CONF_TARGET_NETWORK_DEFAULT_INTERFACE_TYPE != MESH
     Case("UDPSOCKET_ECHOTEST_BURST_NONBLOCK", UDPSOCKET_ECHOTEST_BURST_NONBLOCK),
+#endif
     Case("UDPSOCKET_SENDTO_REPEAT", UDPSOCKET_SENDTO_REPEAT),
     Case("UDPSOCKET_ECHOTEST", UDPSOCKET_ECHOTEST),
+#if MBED_CONF_TARGET_NETWORK_DEFAULT_INTERFACE_TYPE != MESH
     Case("UDPSOCKET_ECHOTEST_BURST", UDPSOCKET_ECHOTEST_BURST),
+#endif
```

Comment:
Same here.. I don't see the point of skipping this.

Reply:
Test skip reverted. It is successful under  local mesh BR. Testing mesh performance under heavy RF traffic is beyond the scope of netsockets tests.